src/shared/ui/date-lang/date-lang.tsx

```diff
@@ -1,31 +1,36 @@
-/* eslint-disable @stylistic/jsx-one-expression-per-line */
+import classNames from 'classnames/bind';
+import { dayJS } from '@/app/services/dayjs';
```

Comment:
you need dayJS only for tests, in other files you you can use dayjs
```suggestion
import dayjs from 'dayjs';
```

Reply:
Thanks for pointing that out.